doc/workshops/scientific-computing.rst

```diff
@@ -2,4 +2,730 @@
 Scientific Computing and Visualization with Spyder
 ##################################################
 
-This is a work in progress. The workshop for Scientific Computing and Visualization with Spyder will be available soon!
+This workshop is a tour of the features that make Spyder an ideal IDE for using the scientific tools offered by Python. Throughout the workshop, we will apply the scientific method to answer some questions related to our preferences and cognitive abilities.
```

Comment:
I think the workshop is not a tour. So maybe something like: This workshop allows you to explore some of Spyder's tools for Scientific computing?